source/cloud-security/office365/monitoring-office365-activity.rst

```diff
@@ -241,32 +241,3 @@ If Wazuh successfully connects to Office 365 API, the events raised above will t
         },
         "location":"office365"
     }
- 
-
-Enabling dashboard visualization  
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Once the configuration is complete, you can activate the corresponding Security Information Management module on the Wazuh Dashboard. This module provides additional details and insights about events, as shown in the screenshots below.
-
-    .. thumbnail:: /images/office365/office365-dashboard.png
-       :title: Office 365 dashboard
-       :alt: Office 365 dashboard
-       :align: center
-       :width: 80%
-
-    .. thumbnail:: /images/office365/office365-events.png
-       :title: Office 365 events
-       :alt: Office 365 events
-       :align: center
-       :width: 80%
```

Comment:
**thought**: this could require rephrasing because the visibility of the module is not configurable. If this was removed, the images could be removed too.

Reply:
Ok, I will remove the related images too.

Reply:
Fixed in ef02e7cd655d1412e5f49256400d4ba4196357cc.